MekHQ/src/mekhq/campaign/market/ContractMarket.java

```diff
@@ -368,14 +368,17 @@ else if (contract.getMissionType() == AtBContract.MT_RIOTDUTY)
 			}
 		}
 
+		int searchRadius = campaign.getCampaignOptions().getSearchRadius();
+		searchRadius += (int)(searchRadius * 0.10 * retries);
```

Comment:
Why multiply the fudge factor by the number of remaining retries? This increases the search radius by 30% the first time, then 20% the second retry, then 10%.
Also, if we're to limit the contract location as well as the employer to the selected search radius, why increase it beyond what the user has set?

Reply:
Doh, that's my misreading of the code.

The intent was to start at the user-specified radius, then cheat upward if no contract locations could be found.